lightstep/resource_metric_dashboard.go

```diff
@@ -64,6 +64,8 @@ func getChartSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Computed: true,
 		},
+		// The y_axis is included here for backwards compatibility, but it's not used anywhere
+		// because it has been removed from Lightstep's public API.
```

Comment:
I'm thinking we can just remove from the schema and cut a new release version that's breaking (bump from 1.5 to 1.6) -- existing clients that haven't upgraded will still work (and will confirm in test).